config/config.php

```diff
@@ -26,7 +26,17 @@
 
 $ver_str = 'Nextcloud Client ' . $ver;
 
-$url = 'https://github.com/nextcloud/desktop/releases/download/v' . $ver . '/';
+if (version_compare($version, '3.0.3') < 0) {
```

Comment:
To clarify: only desktop client 3.0.3 or higher can properly handle the download from GitHub. Thus it is now effectively disabled. That's mainly the root cause for our download server to hit it's bandwidth limits again.

Reply:
Yes, I tried to discuss it with @rullzer yesterday, but I guess he was just busy. ;-)

Reply:
See #45 for an short term "fix".